rewrite-java/src/main/java/org/openrewrite/java/format/TabsAndIndentsVisitor.java

```diff
@@ -401,6 +423,9 @@ private Comment indentComment(Comment comment, int shift) {
             for (char c : comment.getText().toCharArray()) {
                 newTextBuilder.append(c);
                 if (c == '\n') {
+                    // Note: this needs to be adjusted to only pass in whitespace.
```

Comment:
I'm trying to implementing in small chunks. So, I plan to fix this in issue 652.  It _should_ be relatively straight forward ... maybe.